beta/experience-platform-extension/tutorials/tutorial-2-swift-core-setup.md

```diff
@@ -0,0 +1,116 @@
+# Tutorial 2 - Swift Core setup
+
+This tutorial illustrates how you may send XDM commerce events to Adobe Experience Platform via Experience Edge using the AEP Edge extension in a sample application, provided to you in iOS \(Swift\) and Android.
+
+The demo mobile application has multiple tabs. For this exercise, the `Edge` and `Assurance` tabs will be used, demonstrating XDM commerce events in a mobile application.
+
+## Prerequisites for this tutorial
+
+- Access to Adobe Experience Launch dashboard
+- Xcode 12
+- Cocoapod
+
+### Configure the Launch Mobile property
+
+To finish the tutorial, you can either use a existing Mobile Property and create a new one. Follow the below steps if you need to create a new Mobile property in Launch UI:
```

Comment:
should this read "you can either use a existing Mobile Property _or_ create a new one."?